lib/job-iteration/iteration.rb

```diff
@@ -132,6 +134,10 @@ def interruptible_perform(*arguments)
 
       run_callbacks(:shutdown)
 
+      if needs_reenqueue && !completed.nil?
```

Comment:
You can use `completed.present?` here instead of `!completed.nil?`

Reply:
Matt and I chatted about this and it breaks the logic since `completed.present?` isn't _quit_ the same as `!completed.nil?`